src/main/java/fly/wild/wizards/serverside/ServerConnnector.java

```diff
@@ -190,93 +117,208 @@ public boolean configureOneWayTLS () {
 			System.out.println ("Management interface selected");
 			configureManagementInterface ();
 		}
+		reloadServer ();
 			
-
+		System.out.println(this.logMessage.toString());
 		return success;
 	}
 	
 	private void configureUndertow () {
 		
 		/*
+		 * Sample management CLI commands
 		 * /subsystem=undertow/server=default-server/https-listener=https:write-attribute(name=ssl-context, value=examplehttpsSSC) 
 		 */
 	
-		ModelNode undertow = new ModelNode();
-		undertow.get(ClientConstants.OP).set(ClientConstants.WRITE_ATTRIBUTE_OPERATION);
-		undertow.get(ClientConstants.OP_ADDR).add(STRING_SUBSYSTEM, STRING_SUBSYSTEM_UNDERTOW);
-		undertow.get(ClientConstants.OP_ADDR).add(STRING_SERVER_ATTRIBUTE,STRING_SERVER_VALUE);
-		undertow.get(ClientConstants.OP_ADDR).add(STRING_HTTPS_LISTENER_ATTRIBUTE,STRING_HTTPS_LISTENER_VALUE);
-		undertow.get(STRING_NAME_ATTRIBUTE).set(STRING_NAME_VALUE);
-		undertow.get(STRING_VALUE_ATTRIBUTE).set(this.genServerSSLContext);
+		DefaultOperationRequestBuilder undertowBuilder = new DefaultOperationRequestBuilder();
+		undertowBuilder.setOperationName(Util.WRITE_ATTRIBUTE);
+		undertowBuilder.addNode(Util.SUBSYSTEM,Util.UNDERTOW);
+		undertowBuilder.addNode(Util.SERVER,STRING_SERVER_VALUE);
+		undertowBuilder.addNode(Util.HTTPS_LISTENER,STRING_HTTPS_LISTENER_VALUE);
+		undertowBuilder.addProperty(Util.NAME,Util.SSL_CONTEXT);
+		undertowBuilder.addProperty(Util.VALUE,this.genServerSSLContext);
+		System.out.println (undertowBuilder.toString());
 
 		try {
-			this.client = ModelControllerClient.Factory.create (InetAddress.getByName(this.ipAddress), 
-					DMR_PORT);
-			
-			client.execute(new OperationBuilder(undertow).build());
-			
+			ModelControllerClient client = ModelControllerClientFactory.CUSTOM.getClient(connectionAddress, null, false, null, false, 0, null, null, null);
```

Comment:
@theashiot just a minor, connection timeout is set to 0 now, which means no timeout AFAIK. This would be best to change (to for example 3 minutes) once the code is ready for users, as no timeout has a possibility of hanging